### PR TITLE
[Add] Toy Example and Web Application Initializing

### PR DESCRIPTION
**Pull request by 1chooo**
In this change, we initialize the web app with `fastapi`; therefore, we update the `requirements.txt`, also add the toy example of `model` and `web` to proof that the feasibility of this project.

- [x] Initialize web app with `fastapi`
- [x] add dependencies
- [x] toy example of `model` and `web`

Please review, thanks!!!

### CODE REVIEW

File: requirements.txt
Comment (JunTingLin) [inline requirements.txt:1]: dependency issue: tensorflow-io-gcs-filesystem version problem, uvloop does not support Windows

you can: 
- Adjusted tensorflow-io-gcs-filesystem version to >=0.23.1,<0.31.0 to ensure compatibility with current Python version.
- Commented out uvloop as it does not support Windows.

Comment (1chooo) [reply requirements.txt:1]: #4

Comment (1chooo) [reply requirements.txt:1]: @JunTingLin 

第一點，請問要如何修正？若改成 `>=0.23.1,<0.31.0` 將會遇到以下錯誤

```python
ERROR: No matching distribution found for tensorflow-io-gcs-filesystem<0.31.0,==>=0.23.1
```

以下是目前的版本

```python
tensorflow-io-gcs-filesystem==0.37.0
```

Comment (JunTingLin) [reply requirements.txt:1]: @1chooo 

假如我裝的是0.37.0的那版，會遇到以下錯誤:
`ERROR: Ignored the following versions that require a different python version: 0.23.0 Requires-Python >=3.6, <3.10`
`ERROR: Could not find a version that satisfies the requirement tensorflow-io-gcs-filesystem==0.34.0 (from versions: 0.23.1, 0.24.0, 0.25.0, 0.26.0, 0.27.0, 0.28.0, 0.29.0, 0.30.0, 0.31.0)`
`ERROR: No matching distribution found for tensorflow-io-gcs-filesystem==0.37.0`

可能是我們的python版本不一樣，我的python環境版本是`Python 3.10.9`

可以考慮:
1. 較舊版本的 tensorflow-io-gcs-filesystem（例如 0.31.0 我這邊是可以的）
2. 使用相容的 Python 版本（你可以告訴我你的python版本，我跟你裝一樣的)

Comment (1chooo) [reply requirements.txt:1]: OK! 那我改成 `0.31.0`

Comment (1chooo) [reply requirements.txt:1]: 另外我也分一份環境到 `web/`

**Pull request closed without merging**